config/jobs/kubernetes/sig-testing/make-test.yaml

```diff
@@ -48,8 +48,14 @@ presubmits:
     spec:
       # unit tests have no business requiring root or doing privileged operations
       securityContext:
-        runAsUser: 2000
+        # NOTE: these are arbitrary non-root values. They don't exist in the
+        # image and don't need to, the unit tests should only write to TMPDIR
+        runAsUser: 2001
```

Comment:
Just curious: why as this changed? we could just set the runAsGroup to 2000 as well.

Reply:
I thought maybe these are even less likely to be introduced into the images as "real values".

2001 & 2010 is a silly nod.